modules/administration-guide/pages/disabling-readiness-init-containers-for-olm-installer.adoc

```diff
@@ -0,0 +1,6 @@
+[id="disabling-readiness-init-containers-for-olm-installer"]
```

Comment:
Please align ID and file name to the title. Add an alias (redirection) if needed.